retworkx/__init__.py

```diff
@@ -605,9 +605,9 @@ def _graph_dijkstra_shortest_path(
 
 @functools.singledispatch
 def all_pairs_dijkstra_shortest_paths(graph, edge_cost_fn):
-    """Find the shortest path from all nodes
+    """Find the shortest path from all nodes in a graph
```

Comment:
```suggestion
    """For each node in the graph, finds the shortest paths to all others.
```

Just going off the original doc string, but perhaps this would be clearer.